OpenRA.Mods.Common/Orders/PlaceBuildingOrderGenerator.cs

```diff
@@ -210,5 +220,25 @@ public IEnumerable<IRenderable> RenderAfterWorld(WorldRenderer wr, World world)
 		}
 
 		public string GetCursor(World world, CPos xy, MouseInput mi) { return "default"; }
+
+		IEnumerable<Order> TryClearBlockers(World world, CPos topLeft)
```

Comment:
Could this be renamed to reflect its return type, please?
The `Try` prefix is typically reserved for methods such as `TryParse*` and is a `void`.

Perhaps `ClearBlockersOrders`.